contrib/examples/actions/workflows/customfilters/mistral-regex_match.yaml

```diff
@@ -0,0 +1,20 @@
+version: '2.0'
+
+examples.mistral-customfilters-regex_match:
+    description: A workflow for testing regex_match custom filter in mistral
+    type: direct
+    vars:
+        regex_pattern: "([0-9]{3} \\w+ (?:Ave|St|Dr))"
+    input:
+        - input_str
+    output:
+        result_jinja: <% $.result_jinja %>
+        result_yaql: <% $.result_yaql %>
+    tasks:
+
+        task1:
+            action: std.noop
+            publish:
+                result_jinja: '{{ _.input_str | regex_match(_.regex_pattern)}}'
```

Comment:
Calling the filter as normal function call is the minimal requirement IMO. Being able to use the filter from piping is added bonus. Because right now, it's an exception and we will have to document and explain to people the only way to use the function in Jinja is using this format.  I also need to know exactly why it can't be used like normal function calls.